lib/logstash/outputs/firehose.rb

```diff
@@ -45,8 +45,6 @@ class LogStash::Outputs::Firehose < LogStash::Outputs::Base
 
   # make properties visible for tests
   attr_accessor :stream
-  attr_accessor :access_key_id
```

Comment:
how do you specify access and secret keys to your stream from now on?

Reply:
In the same way. By including the AWS Mixin these configuration options are already available.